python/tank/util/shotgun.py

```diff
@@ -226,8 +226,12 @@ def download_url(sg, url, location):
                      to have write permissions
     :raises: :class:`TankError` on failure.
     """
-    # grab proxy server settings from the shotgun API
-    if sg.config.proxy_handler:
+    # We only need to set the auth cookie for downloads from Shotgun server
+    if sg.config.server in url:
```

Comment:
would be great if we could add one docstring example of an sg url that we might download and one of a non-sg (s3) url